cmd/ipfs-cluster-follow/commands.go

```diff
@@ -486,6 +486,57 @@ func listCmd(c *cli.Context) error {
 	return nil
 }
 
+// Stop
+func stopCmd(c *cli.Context) error {
+	clusterName := c.String(clusterNameFlag)
+
+	absPath, configPath, identityPath := buildPaths(c, clusterName)
+	if !isInitialized(absPath) {
+		printNotInitialized(clusterName)
+		return cli.Exit("", 1)
+	}
+
+	// unpin cluster files and kill ipfs-cluster-follow process if cluster and ipfs daemon exist
+	_, err := os.Stat(filepath.Join(absPath, "api-socket"))
+	if err == nil {
+		err = unpinEverything(absPath, clusterName, identityPath)
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "unpin error"), 1)
+		}
+		err = killFollower(absPath)
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "error killing follower peer(ipfs-cluster-follow daemon)"), 1)
+		}
+	} else {
+		return cli.Exit(errors.Wrap(err, "ipfs-cluster-follow or ipfs daemon not exists"), 1)
+	}
+
+	// clean date and folder
+	if c.Bool("cleanup") {
+		cfgHelper, err := cmdutils.NewLoadedConfigHelper(
+			configPath,
+			identityPath,
+		)
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "error creating config helper"), 1)
+		}
+		cfgHelper.Manager().Shutdown()
+		mgr, err := cmdutils.NewStateManagerWithHelper(cfgHelper)
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "error creating state manager"), 1)
+		}
+		err = mgr.Clean()
+		if err != nil {
+			return cli.Exit(errors.Wrap(err, "error cleaning datastore"), 1)
+		}
```

Comment:
If we are removing the whole configuration folder, we might as well not spend time doing `mgr.Clean()` and we can save all of this lines.